package.json

```diff
@@ -16,7 +16,9 @@
     "next-seo": "^4.7.3",
     "react": "16.13.1",
     "react-dom": "16.13.1",
-    "react-reveal": "^1.2.2"
+    "react-reveal": "^1.2.2",
+    "sass": "^1.26.10",
+    "styled-components": "^5.1.1"
```

Comment:
We're not still using styled-components right?

Reply:
Nope, forgot to remove